README.md

```diff
@@ -15,23 +15,23 @@
 
 1. Setup your application at [LinkedIn Developers](https://developer.linkedin.com/).
 
-1. Add `:ueberauth_linkedin` to your list of dependencies in `mix.exs`:
+2. Add `:ueberauth_linkedin` to your list of dependencies in `mix.exs`:
```

Comment:
I think the proper way to do the bullet numbering is like this:

```
1. Foo.
1. Bar.
1. Baz.
```
Then it gets changed by the markdown parser to the correct sequence.